apps/sms/views/shared/js/startup.js

```diff
@@ -100,6 +101,7 @@ var Startup = exports.Startup = {
 
       InterInstanceEventDispatcher.connect();
 
+      MozSettingsClient.init(App.instanceId);
```

Comment:
__nit:__ can we group initialization of all "clients" together, just for the sake of clarity?
```
MozSettingsClient.init(App.instanceId);
MessagingClient.init(App.instanceId);
MozMobileConnectionsClient.init(App.instanceId);

TimeHeaders.init();
Information.initDefaultViews();
ConversationView.init();
```

__question:__ also what do you think if we move perf marks and ```Navigation.setReady``` to result promise callback? To be clearer on the fact that those things don't depend only on ```ConversationView``` initalization.

```
return LazyLoader.load(
  [...this._lazyLoadScripts, ...this._lazyLoadStyles]
).then(() => {
  .....
  return ConversationView.init();
}).then(() => {
  Navigation.setReady();

  window.performance.mark('contentInteractive');
  window.performance.mark('objectsInitEnd');
});
```